src/main/scala/org/allenai/plugins/DockerBuildPlugin.scala

```diff
@@ -112,23 +117,175 @@ object DockerBuildPlugin extends AutoPlugin {
   /** The default copy mapping, set to copy src/main/{bin,conf} to {bin,conf} in the docker image.
     */
   lazy val defaultCopyMappings = Def.setting {
-    val sourceMain = Keys.sourceDirectory.value.toPath.resolve("main")
-    // Copy src/main/{bin,conf} to the staging directory.
-    // See http://www.scala-sbt.org/0.12.3/docs/Detailed-Topics/Mapping-Files.html
-    // for more info on sbt mappings.
-    Seq("bin", "conf").map { dir =>
-      PathFinder(sourceMain.resolve(dir).toFile).***.pair(relativeTo(sourceMain.toFile))
-    }.flatten
+    Seq("bin", "conf").map { dir => (new File(dir), dir) }
+  }
+
+  /** The full image name, derived from the user-provided settings. */
+  lazy val fullImageName: Def.Initialize[String] = Def.setting {
```

Comment:
These are not setting keys - they can't be overridden - and they aren't in `autoImport`, so they won't show up in the `sbt` console. If someone wanted to use them in Scala code, they could do `DockerBuildPlugin.fullImageName`, which I think is qualified-enough without needing the extra `docker` prefix.